Header header

torch_mpc/SteerSetpointKBMState[] trajectory
float32 cost
float32 average_speed
float32 average_curvature
float32[] cost_quantiles
nav_msgs/Odometry odom